charts/posthog/README.md

```diff
@@ -288,3 +290,15 @@ Simply apply one of the following labels to your PR _before merging_ to bump the
 - `bump patch`
 - `bump minor`
 - `bump major`
+
+## Upgrade notes
+
+### Upgrading from 1.x.x
+
+2.0.0 updated redis chart in an incompatible way. If your upgrade fails with
+
+```
+Upgrade "posthog" failed: cannot patch "posthog-posthog-redis-master" with kind StatefulSet: StatefulSet.apps "posthog-posthog-redis-master" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden
+```
+
+Run the following and `helm upgrade` again: `kubectl delete sts posthog-posthog-redis-master`
```

Comment:
does `kubectl delete sts posthog-posthog-redis-master` need a namespace to run? I think it does (just tried it on test DO cluster)